examples/align_sequence.cpp

```diff
@@ -58,8 +58,8 @@ int main(int argc, char *argv[])
 {
 	validateArgs(argc, argv);
 
-	typedef PointMatcher<float> PM;
-	typedef PointMatcherIO<float> PMIO;
+	typedef PointMatcher<double> PM;
+	typedef PointMatcherIO<double> PMIO;
```

Comment:
Does this change affect the performance in any way?

Reply:
I would say that this might affect the performances, but I don't know for sure. I would ask @simonpierredeschenes what are his thoughts about this.

Also, the reason these changes were made is that these examples were used as a reference to determine if the behaviour of the python examples matched that of the C++ examples. So, knowing that the results of the both versions were the same gave me a good indication that what I was doing was right.

Reply:
After a few quick tests, the float version is actually slightly faster than the double version, both for C++ and Python libpointmatcher version. So, I'll change the compile type of the bindings to float and revert the type for the examples as well.

Reply:
Oki. You can still change it easily and re-export it, right?

Can that detail about the Python binding be included in the documentation?

Reply:
> Oki. You can still change it easily and re-export it, right?

Sorry, I'm not sure I'm following you on this one. Do you mean changing the type of the bindings? If so, yes I just need to the type of my alias for the PointMatcher instance.

> Can that detail about the Python binding be included in the documentation?

Yes, I can and just to be sure, the detail you're talking about is the fact that the bindings are compiled using float, right?

Reply:
I meant with my question if it's possible to change the bindings to work with float with minimal overhead.

Yes, mentioning that the work with floats :)